examples/with-javascript/src/commands/General/ping.js

```diff
@@ -12,12 +12,11 @@ class UserCommand extends Command {
 	async messageRun(message) {
 		const msg = await send(message, 'Ping?');
 
-		const content = `Pong from JavaScript! Bot Latency ${Math.round(this.container.client.ws.ping)}ms. API Latency ${
+		const content = `Pong from JavaScript (ESM Mode)! Bot Latency ${Math.round(this.container.client.ws.ping)}ms. API Latency ${
```

Comment:
```suggestion
		const content = `Pong from JavaScript! Bot Latency ${Math.round(this.container.client.ws.ping)}ms. API Latency ${
```

This is not the ESM example